### PR TITLE
docs: add kaplan-meier lesson

### PR DESCRIPTION
**Pull request by jExbrayat**
testt

### CODE REVIEW

File: src/notebooks/non_parametric_estimation.ipynb [hunk 1]
```diff
@@ -106,15 +106,128 @@
     "## Kaplan-Meier"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Formalisation du problème\n",
+    "Soit $\\tau$ une variable aléatoire à valeur dans $\\mathbb{N}^*$ modélisant le jour de succès d'un projet kickstarter.  \n",
```
Comment (Ab2nour) [inline src/notebooks/non_parametric_estimation.ipynb:114]: nit: Kickstarter

File: src/notebooks/non_parametric_estimation.ipynb [hunk 2]
```diff
@@ -106,15 +106,128 @@
...
+    "On se munit d'un échantillon $\\tau_1,\\dots,\\tau_n$ de variables aléatoires indépendantes indentiquement distribuées suivant la même loi que $\\tau$ et de \n",
```
Comment (Ab2nour) [inline src/notebooks/non_parametric_estimation.ipynb:115]: typo: identiquement

File: src/notebooks/non_parametric_estimation.ipynb [hunk 3]
```diff
@@ -106,15 +106,128 @@
...
+    "$$\n",
+    "\\begin{align}\n",
+    "\\hat q(t) & = 1 - \\frac{|\\{1\\le k\\le n\\,:\\, c_k\\ge t,\\tilde \\tau_k=t\\}|}{|\\{1\\le k \\le n\\,:\\, c_k\\ge t,  \\tilde \\tau_k\\ge t\\}|} \\\\\n",
+    "                & = 1 - \\frac{|\\{1\\le k\\le n\\,:\\,\\tilde \\tau_k=t\\}|}{|\\{1\\le k \\le n\\,:\\, \\tilde \\tau_k\\ge t\\}|} \\hspace{12px} \\text{\n",
+    "                        car $c_k \\ge \\tilde \\tau_k = \\min(\\tau_k,c_k)$\n",
+    "                } \\\\\n",
+    "                & = \\frac{d(t)}{n(t)}\n",
+    "\\end{align}\n",
+    "$$\n",
+    "où $d(t)$ est le nombre de succès connues au temps $t$  \n",
```
Comment (Ab2nour) [inline src/notebooks/non_parametric_estimation.ipynb:178]: typo: connu

**Review by Ab2nour — CHANGES_REQUESTED** [review]
Only nit and typo

File: src/notebooks/non_parametric_estimation.ipynb (hunk 1 above)
Comment (jExbrayat) [reply src/notebooks/non_parametric_estimation.ipynb:114]: done

File: src/notebooks/non_parametric_estimation.ipynb (hunk 3 above)
Comment (jExbrayat) [reply src/notebooks/non_parametric_estimation.ipynb:178]: done

**Review by Ab2nour — APPROVED** [review]

**Pull request merged**